### PR TITLE
Remove Flannel in favor of Bridge CNI plugin

### PR DESCRIPTION
**Pull request by oglok**
This PR disables Flannel in favor of the default CNI plugin for our container runtime of choice, CRI-O. The default CNI plugin for CRI-O is the Bridge CNI.

Closes: https://issues.redhat.com/browse/USHIFT-119

### CODE REVIEW

**Comment by oglok** [comment]
https://issues.redhat.com/browse/USHIFT-119

File: packaging/crio.conf.d/microshift.conf [hunk 1]
```diff
@@ -5,10 +5,6 @@ conmon_cgroup = "pod"
 cgroup_manager = "systemd"
 
 [crio.network]
-# cbr0 is the name configured by flannel in /etc/cni/net.d/ config file
-# by declaring this crio will wait until that network is configured.
-cni_default_network = "cbr0"
```
Comment (dhellmann) [inline packaging/crio.conf.d/microshift.conf:10]: Do we need to pick a different default, or is that configured some other way?

Comment (oglok) [reply packaging/crio.conf.d/microshift.conf:10]: We faced an issue with pods starting being attached to two different bridges (CRI-O bridge and Flannel bridge) due to a race condition. We decided to fix it by letting CRI-O wait for the Flannel bridge to be created and attach all pods there. Now that Flannel is gone, we can just rely on the default bridge from CRI-O. 

Short answer, I don't think we need to add any default there.

Comment (dhellmann) [reply packaging/crio.conf.d/microshift.conf:10]: Thanks, that makes sense.

**Review by dhellmann — APPROVED** [review]

**Pull request merged**